source/includes/administration/_organizations.md

```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
 
 Required | &nbsp;
 ---- | ----
-`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
\ No newline at end of file
+`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
+
+<!-------------------- FIND BILLING CYCLE -------------------->
+### Get a billing cycle for an org
+
+`GET /organizations/:organization_id/billing_cycles/:billing_cycle_id`
+
+Retrieves the billing cycle for the organization.
```

Comment:
Retrieves a billing cycle for the organization.
(multiple billing cycles, we fetch one here)

Reply:
4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9